docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -233,42 +251,26 @@ This table contains the data sources for all the measures. The table structure a
      - Sources
      - Description
      - Notes
-   * - prevalence_cid
-     - 
-     - 
-     - 
-   * - birth_prevalence_cid
-     - 
-     - 
-     -
-   * - deaths_cid
-     - 
-     - 
-     - 
-   * - population
-     - 
-     - 
-     - 
-   * - sequelae_cid
-     - 
-     - 
-     - 
-   * - incidence_rate_cid
-     - 
-     - 
-     - 
-   * - remission_rate_mid
-     - 
-     - 
-     - 
-   * - disability_weight_s{`sid`}
-     - 
-     - 
-     - 
-   * - prevalence_s{`sid`}
-     - 
-     - 
-     - 
+   * - prevalence_c426
```

Comment:
In order to transform and load forecast data into vivarium data artifact, can we write down what metric Zach produced for lung cancer forecasts including prevalence, incidence, and deaths.

Reply:
Can you clarify what you mean by this?

Reply:
The measure metric of forecast data provided by Zach is not always the GBD standard rate space. We'd better note the SEs what unit he produced and clarify the necessary transformation if needed. for example the original all-cause mortality rate forecast has a metric of log(acmr) per person-year.